<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectManagerDialog</class>
 <widget class="QDialog" name="ProjectManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>314</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Informational</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTitle">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="descriptionTextEdit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Engineer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="sheetsAreTrackFolderCheckBox">
            <property name="text">
             <string>Sheets behave as Track Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Export</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="sheetsExportButton">
            <property name="text">
             <string>Sheet(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportTemplateButton">
            <property name="text">
             <string>Template</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" rowspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>385</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Sheets</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTreeWidget" name="treeSheetWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>10</horstretch>
             <verstretch>5</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>120</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>Sheet Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tracks</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Length</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Selected Sheet</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="selectedSheetName">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="sheetAudioSourcesFrame">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="sheetAudioSourcesDirLineEdit"/>
               </item>
               <item>
                <widget class="QPushButton" name="sheetAudioSourcesPushButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="deleteSheetButton">
                <property name="text">
                 <string>Delete</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>1</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="renameSheetButton">
                <property name="text">
                 <string>Rename</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>New Sheet</string>
           </property>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>9</number>
            </property>
            <item>
             <layout class="QHBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="createSheetButton">
                <property name="text">
                 <string>Create new Sheet</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cdtext">
      <attribute name="title">
       <string>CD Text</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Performer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditPerformer"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Disc ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditId"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>UPC EAN:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditUPC"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Genre:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxGenre">
         <item>
          <property name="text">
           <string>Unused</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Undefined</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Adult Contemporary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alternative Rock</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Childrens</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Classical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contemporary Christian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Country</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Easy Listening</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Erotic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Folk</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gospel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hip Hop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jazz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Latin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Musical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Age</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Opera</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Operette</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pop Music</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rap</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reggae</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rock Music</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rhythm and Blues</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sound Effects</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spoken Word</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>World Music</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Arranger</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditArranger"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Songwriter</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditSongwriter"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Message</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditMessage"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>42</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Undo last change&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redoButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>42</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Redo last change&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
